src/be_db_txn_actor.erl

```diff
@@ -133,4 +133,11 @@ to_actors(blockchain_txn_payment_v2, T) ->
                        [{"payee", blockchain_payment_v2:payee(Payment)} | Acc]
                end,
     lists:foldl(ToActors, [{"payer", blockchain_txn_payment_v2:payer(T)}],
-                blockchain_txn_payment_v2:payments(T)).
+                blockchain_txn_payment_v2:payments(T));
+to_actors(blockchain_txn_state_channel_open_v1, T) ->
+    %% NOTE: owner => sc_opener, renamed for better reasoning
+    [{"sc_opener", blockchain_txn_state_channel_open_v1:owner(T)} ];
+to_actors(blockchain_txn_state_channel_close_v1, T) ->
+    %% NOTE: closer can be one of the clients of the state channel or the owner of the router
+    %% if the state_channel expires
+    [{"sc_closer", blockchain_txn_state_channel_close_v1:closer(T)} ].
```

Comment:
This is missing a number of actors for the token burn?

Reply:
I don't understand, actors for `blockchain_txn_token_burn_v1` are already defined in previous clauses.

Reply:
I was confusing clauses, ignore